marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/EntitySearchManagerTest.java

```diff
@@ -70,37 +70,29 @@ void searchWithTransformInStagingDatabase() {
     }
 
     @Test
-    void noEntityTypesSelected() {
-        runAsDataHubDeveloper();
-        // Need at least one entity model to exist for this scenario
-        installOnlyReferenceModelEntities(true);
-
-        runAsHubCentralUser();
-
-        SearchQuery query = new SearchQuery();
-        DocSearchQueryInfo info = new DocSearchQueryInfo();
-        info.setEntityTypeIds(Arrays.asList(" "));
-        query.setQuery(info);
+    void searchAllDataInStagingDatabase() {
+        validateAllDataSearch("staging");
+    }
 
-        String results = new EntitySearchManager(getHubClient(), "staging").search(query).get();
-        ObjectNode node = readJsonObject(results);
-        assertEquals(0, node.get("total").asInt(), "When entityTypeIds has values, but they're all empty strings, the " +
-            "backend should return no results, and not throw an error");
+    @Test
+    void searchAllDataInFinalDatabase() {
+        validateAllDataSearch("final");
     }
 
     @Test
     public void testSearchResultsOnNoData() {
         runAsDataHubDeveloper();
-        EntitySearchManager.QUERY_OPTIONS = "non-existent-options";
         String collectionDeleteQuery = "declareUpdate(); xdmp.collectionDelete(\"http://marklogic.com/entity-services/models\")";
         getHubClient().getFinalClient().newServerEval().javascript(collectionDeleteQuery).evalAs(String.class);
 
         runAsHubCentralUser();
-        assertTrue(new EntitySearchManager(getHubClient()).search(new SearchQuery()).get().isEmpty());
+        StringHandle results = new EntitySearchManager(getHubClient()).search(new SearchQuery());
+        ObjectNode node = readJsonObject(results.get());
+        assertEquals(14, node.get("total").asInt());
```

Comment:
Any time there's an assertion on a number, it's helpful to have an assertion message to explain why that number is expected. In this case, you're expecting 14 because that's the number of OOTB DH artifacts. That assertion is a little brittle - if we add a new artifact, this test would break, which would be surprising. 

Instead of assuming 14, do a quick newServerEval and run "cts.estimate(cts.trueQuery())" to get back the count of expected documents. That allows you to make a better assertion - i.e. that you expect the count to be that of all documents in the database (which the user has read access to).